message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -1708,5 +1708,21 @@
       <field type="float" name="min_distance" units="m">Minimum distance the sensor can measure.</field>
       <field type="float" name="max_distance" units="m">Maximum distance the sensor can measure.</field>
     </message>
+    <message id="11038" name="WATER_DEPTH">
+      <wip/>
+      <!-- This message is work-in-progress and it can therefore change and should NOT be used in stable production environments -->
+      <description>Water depth</description>
```

Comment:
maybe need a quality field type to know if we get something/are out of range/get stuck etc.

Reply:
@khancyr thanks for this.  I've added a "healthy" field now.